sdk/OoyalaSkinSDK/sharePanel.js

```diff
@@ -52,19 +53,24 @@ var SharePanel = React.createClass ({
 		this.props.onSocialButtonPress(serviceType);
 	},
 
+	onSocialAlertDismiss: function() {
+		Log.log('Ok Pressed!');
+		this.props.onSocialAlertDismiss();
+	},
+
 	render: function() {
 		var sharePanel;
 		var socialButtons = [];
 
 		var postShareSuccessAlert;
 
-		if(this.props.alertTitle != ""){
+		if(this.props.alertTitle !== ""){
 			postShareSuccessAlert = AlertIOS.alert(
 				this.props.alertTitle,
-  			this.props.alertMessage,
-  			[
-    			{text: 'Ok', onPress: () => Log.log('Ok Pressed!')},
-  			]
+				this.props.alertMessage,
+				[
+					{ text: 'Ok', onPress: this.onSocialAlertDismiss },
```

Comment:
is there any way to click out of the box (in other words, not click on the "Ok" button)?